src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusListener.cs

```diff
@@ -302,11 +301,13 @@ internal void StartMessageBatchReceiver(CancellationToken cancellationToken)
                                     }
                                 }
                             }
-
-                            // Close the session and release the session lock
-                            if (_isSessionsEnabled)
+                            else

```

Comment:
>else [](start = 28, length = 4)

Should this else come after another `}`? Or I might be counting wrong. I assume the intention is to close the receiver only when `messages == null`

Reply:
Good catch, I just counted and it does need to be one level of nesting out (after line 312)